src/js/views/forms/form/form_views.js

```diff
@@ -169,12 +176,31 @@ const LayoutView = View.extend({
       replaceElement: false,
     },
     status: '[data-status-region]',
+    widgets: '[data-widgets-region]',
+  },
+  templateContext() {
+    return {
+      formHasWidgets: this.formHasWidgets,
+    };
+  },
+  initialize({ patient, widgets }) {
+    this.patient = patient;
+    this.widgets = widgets;
+    this.formHasWidgets = widgets && widgets.length;
   },
   onRender() {
     this.showChildView('contextTrail', new ContextTrailView({
       patient: this.getOption('patient'),
       action: this.getOption('action'),
     }));
+
+    if (this.formHasWidgets) {
+      this.showChildView('widgets', new WidgetCollectionView({
```

Comment:
I think you could add a `className: 'flex-wrap',` here to cover the style issue.  Or if every instance of `WidgetCollectionView` would benefit from the class that same line can be added to it's definition in the view file.